### PR TITLE
pagination

### PR DESCRIPTION
**Pull request by judymou**

### CODE REVIEW

**Comment by judymou** [comment]
https://yufsmjnrpg.localtunnel.me

File: controllers/directory.js [hunk 1]
```diff
@@ -4,6 +4,26 @@ function getStaticMapUrl(lat, lng) {
   return 'http://maps.googleapis.com/maps/api/staticmap?zoom=2&scale=1&size=350x250&maptype=terrain&format=png&visual_refresh=true&markers=size:mid%7Ccolor:red%7Clabel:1%7C' + lat + ',' + lng + '&key=' + process.env.MAPS_API_KEY;
 }
 
+function getPaginationData(currentPage, totalPage) {
+  currentPage = parseInt(currentPage, 10) || 1;
+  // Display 5 pages at once.
+  var NUM_PAGES = 5;
```
Comment (typpo) [inline controllers/directory.js]: You can move this outside the function because it's a constant

File: controllers/directory.js [hunk 2]
```diff
@@ -4,6 +4,26 @@ function getStaticMapUrl(lat, lng) {
...
+  var startPage = Math.max(1, currentPage - Math.floor(NUM_PAGES / 2));
+  if (startPage > totalPage - NUM_PAGES + 1) {
+    startPage = totalPage - NUM_PAGES + 1;
+  }
+
+  console.log(startPage);
```
Comment (typpo) [inline controllers/directory.js]: Remove console log

File: controllers/directory.js [hunk 3]
```diff
@@ -4,6 +4,26 @@ function getStaticMapUrl(lat, lng) {
...
+  console.log(startPage + NUM_PAGES);
+  console.log(totalPage);
+  return {currentPage: currentPage,
```
Comment (typpo) [inline controllers/directory.js]: move first key onto newline

File: views/partials/pagination.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<ul id="pagination" class="pagination-sm pagination">
```
Comment (typpo) [inline views/partials/pagination.html]: ❤️

File: controllers/directory.js (hunk 2 above)
Comment (judymou) [inline controllers/directory.js]: removed

File: controllers/directory.js (hunk 3 above)
Comment (judymou) [inline controllers/directory.js]: done

**Pull request merged**